addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1575,7 +1575,7 @@ const Wysiwyg = Widget.extend({
                         // Unstash the mutations now that the color is picked.
                         this.odooEditor.historyUnstash();
                         this._processAndApplyColor(eventName, ev.data.color);
-                        this._updateEditorUI();
+                        this._updateEditorUI(this.lastMediaClicked ? { target: this.lastMediaClicked } : undefined);
```

Comment:
What about `this.lastMediaClicked && { target: this.lastMediaClicked }` ? If `this.lastMediaClicked` is undefined then it will return `undefined` and that is what you want.

Reply:
Sure indeed.